packages/thirdweb/src/wallets/connection/autoConnect.ts

```diff
@@ -0,0 +1,58 @@
+import { webLocalStorage } from "../../utils/storage/webStorage.js";
+import { createWallet } from "../create-wallet.js";
+import { getDefaultWallets } from "../defaultWallets.js";
+import { getInstalledWalletProviders } from "../injected/mipdStore.js";
+import type { Wallet } from "../interfaces/wallet.js";
+import type { ConnectionManager } from "../manager/index.js";
+import { autoConnectCore } from "./autoConnectCore.js";
+import type { AutoConnectProps } from "./types.js";
+
+/**
+ * Attempts to automatically connect to a wallet based on the provided configuration.
```

Comment:
What about something like "Attempts to automatically connect to the last connected wallet."